docs/source/upcoming_release_notes/619-New_Lcls_object_to_query_machine_status.rst

```diff
@@ -0,0 +1,31 @@
+619 New Lcls object to query machine status
+###########################################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A 
+
+Device Updates
+--------------
+- The `SxrGmD` device has been removed from `beam_stats` module. SXR has been disassembled and the GMD was moved into the EBD. Its MJ PVs was not working anymore.
```

Comment:
This counts as an API change since someone who tries to import this device will find it missing

Reply:
ok i'll update that, thank you